src/x86/cdef16_avx2.asm

```diff
@@ -2,12 +2,26 @@
 ; Copyright (c) 2021, Nathan Egge
 ; All rights reserved.
 ;
-; This source code is subject to the terms of the BSD 2 Clause License and
-; the Alliance for Open Media Patent License 1.0. If the BSD 2 Clause License
-; was not distributed with this source code in the LICENSE file, you can
-; obtain it at www.aomedia.org/license/software. If the Alliance for Open
-; Media Patent License 1.0 was not distributed with this source code in the
-; PATENTS file, you can obtain it at www.aomedia.org/license/patent.
+; Redistribution and use in source and binary forms, with or without
```

Comment:
@negge are you ok with the re-licensing?

Reply:
@negge consented and applied the boilerplate change for acceptance upstream. He has affirmed replacing with the upstream version, in discussion of this PR on IRC.

Reply:
Yes, and licenses between the projects are compatible anyway.